### PR TITLE
Sanitize HTML and enable more GitHub markdown.

### PR DESCRIPTION
**Pull request by isoos**
#1418

### CODE REVIEW

File: app/lib/shared/markdown.dart [hunk 1]
```diff
@@ -6,28 +6,22 @@ import 'package:logging/logging.dart';
...
-void _initInlineSyntaxes() {
-  if (_inlineSyntaxes != null) return;
-  _inlineSyntaxes = m.ExtensionSet.gitHubWeb.inlineSyntaxes
-      .where((s) => s is! m.InlineHtmlSyntax)
-      .toList();
-  _inlineSyntaxes.add(_InlineBrHtmlSyntax());
-}
+const _whitelistedClassNames = <String>[
+  'hash-header',
+  'hash-link',
```
Comment (jonasfj) [inline app/lib/shared/markdown.dart]: How about `language-dart` and `language-yaml`.... maybe even `language-<whatever>` to allow for syntax highlighting?

Comment (isoos) [reply app/lib/shared/markdown.dart]: Good catch! Coincidentally (as we were talking about tests), this was caught by the templates_test.dart too :)

Comment (kevmoo) [reply app/lib/shared/markdown.dart]: FYI – could be a Set instead of a list 😄

Comment (isoos) [reply app/lib/shared/markdown.dart]: ah, sorry, missed this comment... not sure though if it will be faster with only two items in it...

Comment (kevmoo) [reply app/lib/shared/markdown.dart]: 🤷‍♂

**Pull request merged**